ci-operator/config/kubevirt/kubevirt/kubevirt-kubevirt-master.yaml

```diff
@@ -29,49 +21,67 @@ tag_specification:
   namespace: ocp
 tests:
 - artifact_dir: /go/src/kubevirt.io/kubevirt/_out/exported-artifacts
-  as: e2e-periodic-azure-nested-virt
-  commands: |
-    set -euo pipefail
+  as: e2e-nested-virt
+  cron: 2 3 * * *
+  steps:
+    cluster_profile: azure4
+    test:
+    - as: nightly-build-azure4
+      commands: |
+        set -euo pipefail
 
-    release_base_url="https://gcsweb.apps.ovirt.org/gcs/kubevirt-prow/devel/nightly/release/kubevirt/kubevirt"
-    release_date=$(curl -L "${release_base_url}/latest")
-    release_url="${release_base_url}/${release_date}"
-    commit=$(curl -L "${release_url}/commit")
+        echo "Downloading oc binary"
+        # FIXME: transfer into Dockerfile for ci
```

Comment:
is this a later to do?

Reply:
Actually I'm not really sure what to do with this. The DPTP team told me they are working on this to not leave the tedious download burden on the user, so this is more of a reminder for me to nag there sometime later ;-)